debian/asterisk.postinst

```diff
@@ -90,8 +90,15 @@ case "$1" in
 	done
 
     # WAZO: Do a more aggressive chmod/chown on asterisk files
-    asterisk_dirs="/etc/asterisk /var/lib/asterisk /var/spool/asterisk/monitor"
-
+    asterisk_dirs="/etc/asterisk /var/lib/asterisk /var/spool/asterisk/monitor /var/lib/asterisk/moh /var/lib/asterisk/sounds/custom /usr/share/asterisk/moh /var/spool/asterisk/fax"
+    # NOTE(afournier): originally missing dirs:
+    # /var/lib/asterisk/moh
+    # /var/lib/asterisk/sounds/custom
+    # /usr/share/asterisk/moh
```

Comment:
Can wazo-confd still manage sound files, moh with this change?

Reply:
I tried:
- Voicemails
  - Greetings read, add and delete through Portal
  - Same thing through phone (AGI)
  - Record, read, delete voicemail through AGI
  - Record, read, delete voicemail through app (calld)
- Music on hold add/read/delete
- Sound file add/read/delete
- Recording add/read/delete
   - Record through *9
   - Read and delete through Portal
  
If I missed anything let me know and I will test it